int16 can_id
string name
int16 data